indico/MaKaC/webinterface/tpls/CategoryDisplay.tpl

```diff
@@ -93,12 +93,16 @@ containsCategories = len(categ.getSubCategoryList()) > 0
     % if isRootCategory or categ.attached_items or managers or allowUserModif:
     <div class="category-sidebar">
         % if isRootCategory:
-            % if isNewsActive:
+            <%
+            from indico.modules.news.util import get_recent_news
+            news = get_recent_news()
+            %>
+            % if news:
                 <h2 class="icon-bullhorn">
                     ${_("News")}
-                    <a href="${ urlHandlers.UHIndicoNews.getURL()}" class="more-icon">${ _("more...") }</a>
+                    <a href="${url_for('news.display')}" class="more-icon">${ _("show all...") }</a>
```

Comment:
Here you didn't leave a space before `url_for`, but later on in `indico/MaKaC/webinterface/tpls/WelcomeHeader.tpl` line 6 you do.